test/integration/action/action-integration.spec.js

```diff
@@ -140,12 +141,14 @@ describe('Action Integration Tests', function() {
 
     await grcpClient.init({
       ipcMain: ipcMainStub1,
+      lndIP,
       lndPort: LND_PORT_1,
       lndSettingsDir: LND_SETTINGS_DIR_1,
       network: NETWORK,
     });
     await grcpClient.init({
       ipcMain: ipcMainStub2,
+      lndIP,
```

Comment:
Changes to this file shouldn't be necessary if the default value is set in `grpc-client.js`.